components/apimgt/org.wso2.carbon.apimgt.rest.api.webserver/src/gen/java/org/wso2/carbon/apimgt/rest/api/webserver/PublisherApi.java

```diff
@@ -34,30 +36,32 @@
     service = Microservice.class,
     immediate = true
 )
+//@Path(".[\\d]+/publisher")
```

Comment:
Remove the commented code block

Reply:
Fixed with https://github.com/RenukaFernando/carbon-apimgt/commit/88634c560fb8dfbf87211a105d87445605d8592f